src/Interfaces/Options/Particles/IDensity.ts

```diff
@@ -0,0 +1,10 @@
+export interface IDensity {
+    enable: boolean;
+
+    /**
+     * @deprecated use the new area instead
+     */
+    value_area: number;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'value_area' is not in camel case.](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5136254)